src/components/header/style.module.less

```diff
@@ -354,10 +354,17 @@
 	}
 }
 
+.socialItem {
+	padding: 0 0.5rem;
+}
+
+.translation button {
+	padding: 0 0.6rem 0 0.5rem;
```

Comment:
0.5 is the standard padding for the social icons, whilst 0.6 is what the 'About' dropdown menu uses.

Therefore we use 0.5 on the left to keep it a bit tighter/more consistent with the social icons, but 0.6 on the right to give the dropdown arrow more breathing room.